examples/config_files/settings/settings_user.rst

```diff
@@ -1,6 +1,250 @@
 .. _examples_config_files_settings_user:
 
-Using settings_user.yml
-=======================
+Customize your settings: create your settings_user.yml
+======================================================
 
-TBD
+Please, first of all, clone the sources to recreate this project. You can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ in GitHub:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples2.git
+    $ cd examples2/examples/config_files/settings
```

Comment:
Use a ``settings_user`` subfolder, don't get the main scope for this, because other examples related to settings might happen too, better ``examples2/examples/config_files/settings_user``